libs/core/src/scss/interaction-state/_interaction-state.utilities.scss

```diff
@@ -4,7 +4,7 @@
 
 $default-loudness-hover: 'xxs';
 $default-loudness-active: 'xs';
-$default-transition-duration: utils.get-transition-duration('quick');
+$default-transition-duration: 80ms;
```

Comment:
Consider adding duration to transition vars utils.get-transition-duration('insanely-quick');